src/test/java/core/basesyntax/service/file/impl/FileWriterTest.java

```diff
@@ -0,0 +1,55 @@
+package core.basesyntax.service.file.impl;
+
+import core.basesyntax.fruitoperation.Operation;
+import core.basesyntax.fruitoperation.OperationBalance;
+import core.basesyntax.fruitoperation.OperationPurchase;
+import core.basesyntax.fruitoperation.OperationReturn;
+import core.basesyntax.fruitoperation.OperationSupply;
+import core.basesyntax.fruitoperation.Operations;
+import core.basesyntax.fruitoperation.strategy.OperationStrategy;
+import core.basesyntax.fruitoperation.strategy.OperationStrategyImpl;
+import core.basesyntax.service.FruitService;
+import core.basesyntax.service.FruitServiceImpl;
+import core.basesyntax.service.file.DataReader;
+import core.basesyntax.service.file.DataWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class FileWriterTest {
+    private static FruitService fruitService;
```

Comment:
let's not depend on other services when we want to check the particular one

Reply:
done